feeds/transactions_test.go

```diff
@@ -68,24 +68,31 @@ func TestTransactionFeed_ForEachTransaction(t *testing.T) {
 }
 
 func TestTransactionFeed_ToMessage(t *testing.T) {
-	bf := NewMockBlockFeed([]*domain.BlockEvent{
-		{
-			EventType:  domain.EventTypeBlock,
-			Block:      testutils.TestBlock(1),
-			Timestamps: &domain.TrackingTimestamps{Block: time.Now().UTC()},
-		},
-	})
+	var blockEvents []*domain.BlockEvent
+	for i := 0; i < 1000; i++ {
+		blockEvents = append(
+			blockEvents, &domain.BlockEvent{
+				EventType:  domain.EventTypeBlock,
+				Block:      testutils.TestBlock(i),
+				Timestamps: &domain.TrackingTimestamps{Block: time.Now().UTC()},
+			},
+		)
+	}
+	bf := NewMockBlockFeed(blockEvents)
 
 	txFeed, _ := getTestTransactionFeed(t, bf)
 
 	var result *domain.TransactionEvent
-	err := txFeed.ForEachTransaction(func(evt *domain.BlockEvent) error { return nil }, func(evt *domain.TransactionEvent) error {
-		result = evt
-		return nil
-	})
+	err := txFeed.ForEachTransaction(
```

Comment:
Do we need to check this error?

Reply:
resolved: 010dfa60012984dbf6ff74f7a182835371acec72